resources/views/welcome.blade.php

```diff
@@ -5,14 +5,14 @@
         <div class="container home-search-container pt-5 pb-5">
             <h1 class="text-center">Search Classifieds</h1>
             <p class="text-center lead">Search from over {ADS COUNT} classifieds & Post unlimited classifieds free!</p>
-            <form action="#" method="post" class="home-search-form">
+            <form action="{{route('searchlisting')}}" method="get" class="home-search-form">
```

Comment:
Method should be post.

Reply:
No, the query should be visible with domain name.